shared_libraries/orca_shared/database/adapters/api/aws.py

```diff
@@ -57,7 +60,7 @@ def get_configuration(db_connect_info_secret_arn: str, logger: logging.Logger) \
             "Successfully retrieved db login info for both user and admin as a dictionary."
         )
     except Exception:
-        logger.critical("Failed to retrieve secret.", exc_info=True)
+        logger.critical("Failed to retrieve secret.")
```

Comment:
Why was exc_info removed?

Reply:
exc_info is not present in powertools logger like the normal logging library in python.

Reply:
changed to LOGGER.exception